sim/src/test/scala/midasexamples/TutorialSuite.scala

```diff
@@ -122,6 +123,41 @@ abstract class TutorialSuite(
     }
   }
 
+  /**
+    * Compares an AutoCounter output CSV against a reference generated using in-circuit printfs.
+    */
+  def checkAutoCounterCSV(filename: String, stdoutPrefix: String) {
+    it should s"produce a csv file (${filename}) with the correct content" in {
```

Comment:
```suggestion
    it should s"produce a csv file (${filename}) that matches in-circuit printf output" in {
```
don't hide what you mean by correctness.